perf/perf.ml

```diff
@@ -0,0 +1,18 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *         Copyright INRIA, CNRS and contributors             *)
+(* <O___,, * (see version control and CREDITS file for authors & dates) *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+
+external init : unit -> unit = "CAML_init"
+external drop : unit -> unit = "CAML_drop"
+external peek : unit -> int  = "CAML_peek"
+
+let init =
+  if Sys.word_size = 64 then init else
+  (* Instruction counts easily overflow on 32-bits architectures. *)
+  fun _ -> failwith "not supported"
```

Comment:
Do we care that 32-bit Linux systems are not supported?

If yes, we can use `Int64.t` instead of `int`.

Reply:
Switched to `Int64.t`, since using an extra boxed term won't make a huge difference here.